play-json/shared/src/main/scala/Json.scala

```diff
@@ -414,6 +419,29 @@ object Json extends JsonFacade {
   sealed trait MacroOptions
 
   /**
+   * Defines the default macro options.
+   *
+   * Since you can't have defaults for a type parameter (unless it's a contravariant type, it will default to
+   * Nothing), we supply the default via an implicit parameter.
+   */
+  trait DefaultMacroOptions[O <: Json.MacroOptions]
+
+  /**
+   * Low priority implicit used when Json.MacroOptions is not passed.
```

Comment:
For me this new type doesn't represent the default options themselves, but the evidence that passed options conform with option type.

Would personally prefer a more neutral name (`Aux` ?)